<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>MetaLab &#8211; iMetaWiki</title>
	<atom:link href="https://ningzhibin.github.io/category/news-updates/metalab-news-updates/feed/" rel="self" type="application/rss+xml" />
	<link>http://ningzhibin.github.io/</link>
	<description>All you need to know about the iMetaLab suite</description>
	<lastBuildDate>Mon, 27 Jan 2020 18:05:33 +0000</lastBuildDate>
	<language>en-US</language>
	<sy:updatePeriod>hourly</sy:updatePeriod>
	<sy:updateFrequency>1</sy:updateFrequency>
	<generator>https://wordpress.org/?v=5.0.3</generator>

<image>
	<url>https://ningzhibin.github.io/wp-content/uploads/2019/01/logo_M-1.png</url>
	<title>MetaLab &#8211; iMetaWiki</title>
	<link>http://ningzhibin.github.io/</link>
	<width>32</width>
	<height>32</height>
</image> 
	<item>
		<title>bugfix and update: 20190911</title>
		<link>https://ningzhibin.github.io/2019/09/17/bugfix-and-update-20190911/</link>
		<comments>https://ningzhibin.github.io/2019/09/17/bugfix-and-update-20190911/#respond</comments>
		<pubDate>Tue, 17 Sep 2019 18:05:07 +0000</pubDate>
		<dc:creator><![CDATA[Zhibin]]></dc:creator>
				<category><![CDATA[MetaLab]]></category>
		<category><![CDATA[News & Updates]]></category>

		<guid isPermaLink="false">https://ningzhibin.github.io/?p=1334</guid>
		<description><![CDATA[resource config path issue TMT issue console output: task finish marking Page Views: 42]]></description>
				<content:encoded><![CDATA[
<ul><li>resource config path issue</li><li>TMT issue</li><li>console output: task finish marking</li></ul>
<div class="post-views post-1334 entry-meta">
			<span class="post-views-icon dashicons dashicons-chart-bar"></span>
			<span class="post-views-label">Page Views: </span>
			<span class="post-views-count">42</span>
			</div>]]></content:encoded>
			<wfw:commentRss>https://ningzhibin.github.io/2019/09/17/bugfix-and-update-20190911/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>An innovative way for experiment setting</title>
		<link>https://ningzhibin.github.io/2019/09/08/an-innovative-way-for-experiment-setting/</link>
		<comments>https://ningzhibin.github.io/2019/09/08/an-innovative-way-for-experiment-setting/#respond</comments>
		<pubDate>Sun, 08 Sep 2019 15:25:53 +0000</pubDate>
		<dc:creator><![CDATA[Zhibin]]></dc:creator>
				<category><![CDATA[iMetaLab]]></category>
		<category><![CDATA[MetaLab]]></category>
		<category><![CDATA[News & Updates]]></category>

		<guid isPermaLink="false">https://ningzhibin.github.io/?p=947</guid>
		<description><![CDATA[We have designed an effective way to set up the experiment. In MaxQuant, we have to fill Experiment column one-by-one or using Excel to edit the saved template and import it after. Here in MetaLab, you can just simply set...<br /><a class="read-more-button" href="https://ningzhibin.github.io/2019/09/08/an-innovative-way-for-experiment-setting/">Read more</a>]]></description>
				<content:encoded><![CDATA[
<p>We have designed an effective way to set up the experiment.  In MaxQuant, we have to fill Experiment column one-by-one or using Excel to edit the saved template and import it after. Here in MetaLab, you can just simply set the separator and choose the useful pieces of information to form the Experiment.  </p>



<figure class="wp-block-image"><img src="https://ningzhibin.github.io/wp-content/uploads/2019/09/experiment_setting-1-1024x266.png" alt="" class="wp-image-948" srcset="https://ningzhibin.github.io/wp-content/uploads/2019/09/experiment_setting-1-1024x266.png 1024w, https://ningzhibin.github.io/wp-content/uploads/2019/09/experiment_setting-1-300x78.png 300w, https://ningzhibin.github.io/wp-content/uploads/2019/09/experiment_setting-1-768x200.png 768w, https://ningzhibin.github.io/wp-content/uploads/2019/09/experiment_setting-1-956x248.png 956w, https://ningzhibin.github.io/wp-content/uploads/2019/09/experiment_setting-1.png 1189w" sizes="(max-width: 1024px) 100vw, 1024px" /></figure>



<p>

If you are not satisfied with the Experiment name set automatically, you are still able to edit in-cell by double-clicking it. 

</p>
<div class="post-views post-947 entry-meta">
			<span class="post-views-icon dashicons dashicons-chart-bar"></span>
			<span class="post-views-label">Page Views: </span>
			<span class="post-views-count">32</span>
			</div>]]></content:encoded>
			<wfw:commentRss>https://ningzhibin.github.io/2019/09/08/an-innovative-way-for-experiment-setting/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Open-search for PTM now in Metalab</title>
		<link>https://ningzhibin.github.io/2019/09/05/open-search-for-ptm-now-in-metalab/</link>
		<comments>https://ningzhibin.github.io/2019/09/05/open-search-for-ptm-now-in-metalab/#respond</comments>
		<pubDate>Thu, 05 Sep 2019 02:43:06 +0000</pubDate>
		<dc:creator><![CDATA[Zhibin]]></dc:creator>
				<category><![CDATA[MetaLab]]></category>
		<category><![CDATA[News & Updates]]></category>
		<category><![CDATA[Related Publications]]></category>
		<category><![CDATA[opensearch]]></category>
		<category><![CDATA[PTM]]></category>

		<guid isPermaLink="false">https://ningzhibin.github.io/?p=565</guid>
		<description><![CDATA[Our work is just online in bioRxiv https://www.biorxiv.org/content/10.1101/753996v1 Abstract Studying the structure and function of microbiomes is an emerging research field. Metaproteomic approaches focusing on the characterization of expressed proteins and post-translational modifications (PTM) provide deeper understanding of microbial communities....<br /><a class="read-more-button" href="https://ningzhibin.github.io/2019/09/05/open-search-for-ptm-now-in-metalab/">Read more</a>]]></description>
				<content:encoded><![CDATA[
<p>Our work is just online in bioRxiv  <br><a href="https://www.biorxiv.org/content/10.1101/753996v1">https://www.biorxiv.org/content/10.1101/753996v1</a> </p>



<p></p>



<h2>Abstract</h2>



<p>Studying the structure and function of microbiomes is an emerging research field. Metaproteomic approaches focusing on the characterization of expressed proteins and post-translational modifications (PTM) provide deeper understanding of microbial communities. Previous research has highlighted the value of examining microbiome-wide protein expression in studying the roles of the microbiome in human diseases. Nevertheless, regulation of protein functions in complex microbiomes remains under-explored. This is mainly due to the lack of efficient bioinformatics tools to identify and quantify PTMs in the microbiome. Here we implemented an unrestricted open search strategy and developed an advanced multistage post-validation workflow for the identification of PTMs with high accuracy and sensitivity. This approach is applicable to proteomic and metaproteomic datasets. For metaproteomics, this method was combined into a complete data post-analysis workflow, including a quantitative module, to provide a novel and comprehensive data processing platform for metaproteomic analysis. This bioinformatics platform provides information about proteins, PTMs, taxa, functions, and pathways of microbial communities. The performance of the workflow was evaluated using conventional proteomics, metaproteomics from mouse and human gut microbiomes, and modification-specific enriched datasets.</p>
<div class="post-views post-565 entry-meta">
			<span class="post-views-icon dashicons dashicons-chart-bar"></span>
			<span class="post-views-label">Page Views: </span>
			<span class="post-views-count">60</span>
			</div>]]></content:encoded>
			<wfw:commentRss>https://ningzhibin.github.io/2019/09/05/open-search-for-ptm-now-in-metalab/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
	</channel>
</rss>
