<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>iMetaWiki</title>
	<atom:link href="https://ningzhibin.github.io/feed/" rel="self" type="application/rss+xml" />
	<link>http://ningzhibin.github.io/</link>
	<description>All you need to know about the iMetaLab suite</description>
	<lastBuildDate>Mon, 27 Jan 2020 18:05:33 +0000</lastBuildDate>
	<language>en-US</language>
	<sy:updatePeriod>hourly</sy:updatePeriod>
	<sy:updateFrequency>1</sy:updateFrequency>
	<generator>https://wordpress.org/?v=5.0.3</generator>

<image>
	<url>https://ningzhibin.github.io/wp-content/uploads/2019/01/logo_M-1.png</url>
	<title>iMetaWiki</title>
	<link>http://ningzhibin.github.io/</link>
	<width>32</width>
	<height>32</height>
</image> 
	<item>
		<title>bugfix and update: 20190911</title>
		<link>https://ningzhibin.github.io/2019/09/17/bugfix-and-update-20190911/</link>
		<comments>https://ningzhibin.github.io/2019/09/17/bugfix-and-update-20190911/#respond</comments>
		<pubDate>Tue, 17 Sep 2019 18:05:07 +0000</pubDate>
		<dc:creator><![CDATA[Zhibin]]></dc:creator>
				<category><![CDATA[MetaLab]]></category>
		<category><![CDATA[News & Updates]]></category>

		<guid isPermaLink="false">https://ningzhibin.github.io/?p=1334</guid>
		<description><![CDATA[resource config path issue TMT issue console output: task finish marking]]></description>
				<content:encoded><![CDATA[
<ul><li>resource config path issue</li><li>TMT issue</li><li>console output: task finish marking</li></ul>
]]></content:encoded>
			<wfw:commentRss>https://ningzhibin.github.io/2019/09/17/bugfix-and-update-20190911/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>How to upgrade from 1.0 to 2.0?</title>
		<link>https://ningzhibin.github.io/2019/09/17/how-to-upgrade-from-1-0-to-2-0/</link>
		<comments>https://ningzhibin.github.io/2019/09/17/how-to-upgrade-from-1-0-to-2-0/#respond</comments>
		<pubDate>Tue, 17 Sep 2019 03:11:21 +0000</pubDate>
		<dc:creator><![CDATA[Zhibin]]></dc:creator>
				<category><![CDATA[FAQs]]></category>
		<category><![CDATA[iMetaLab]]></category>
		<category><![CDATA[MetaLab]]></category>

		<guid isPermaLink="false">https://ningzhibin.github.io/?p=1331</guid>
		<description><![CDATA[If you are users of MetaLab desktop 1.0, it is easy to upgrade. Just download the 2.0 and override everything from 1.0. All the databases, including the fasta, pept2taxa, and taxon annotations still work in 2.0.]]></description>
				<content:encoded><![CDATA[
<p>If you are users of MetaLab desktop 1.0, it is easy to upgrade. Just download the 2.0 and override everything from 1.0. All the databases, including the fasta, pept2taxa, and taxon annotations still work in 2.0.</p>
]]></content:encoded>
			<wfw:commentRss>https://ningzhibin.github.io/2019/09/17/how-to-upgrade-from-1-0-to-2-0/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>New Publication using Metalab on Nature communication: An in vitro model maintaining taxon-specificfunctional activities of the gut microbiome</title>
		<link>https://ningzhibin.github.io/2019/09/12/new-publication-using-metalab-on-nature-communication-an-in-vitro-model-maintaining-taxon-specificfunctional-activities-of-the-gut-microbiome/</link>
		<comments>https://ningzhibin.github.io/2019/09/12/new-publication-using-metalab-on-nature-communication-an-in-vitro-model-maintaining-taxon-specificfunctional-activities-of-the-gut-microbiome/#respond</comments>
		<pubDate>Thu, 12 Sep 2019 13:37:08 +0000</pubDate>
		<dc:creator><![CDATA[Zhibin]]></dc:creator>
				<category><![CDATA[Related Publications]]></category>

		<guid isPermaLink="false">https://ningzhibin.github.io/?p=1200</guid>
		<description><![CDATA[]]></description>
				<content:encoded><![CDATA[
<figure class="wp-block-image"><img src="https://ningzhibin.github.io/wp-content/uploads/2019/09/nc_leyuan.jpg" alt="" class="wp-image-1201" srcset="https://ningzhibin.github.io/wp-content/uploads/2019/09/nc_leyuan.jpg 895w, https://ningzhibin.github.io/wp-content/uploads/2019/09/nc_leyuan-300x261.jpg 300w, https://ningzhibin.github.io/wp-content/uploads/2019/09/nc_leyuan-768x668.jpg 768w" sizes="(max-width: 895px) 100vw, 895px" /></figure>
]]></content:encoded>
			<wfw:commentRss>https://ningzhibin.github.io/2019/09/12/new-publication-using-metalab-on-nature-communication-an-in-vitro-model-maintaining-taxon-specificfunctional-activities-of-the-gut-microbiome/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>What is the relationship between iMetalab membersï¼Ÿ</title>
		<link>https://ningzhibin.github.io/2019/09/11/what-is-the-relationship-between-imetalab-members%ef%bc%9f/</link>
		<comments>https://ningzhibin.github.io/2019/09/11/what-is-the-relationship-between-imetalab-members%ef%bc%9f/#respond</comments>
		<pubDate>Wed, 11 Sep 2019 15:11:27 +0000</pubDate>
		<dc:creator><![CDATA[Zhibin]]></dc:creator>
				<category><![CDATA[FAQs]]></category>
		<category><![CDATA[iMetaLab]]></category>
		<category><![CDATA[MetaLab]]></category>

		<guid isPermaLink="false">https://ningzhibin.github.io/?p=1190</guid>
		<description><![CDATA[It might be a bit confusing, because of historical reasons. Metalab was initially developed as a desktop software to wrap up MetaProIQ workflow. Upon more request, a cloud version was then developed for general users, called iMetaLab Cloud. Therefore Metalab...<br /><a class="read-more-button" href="https://ningzhibin.github.io/2019/09/11/what-is-the-relationship-between-imetalab-members%ef%bc%9f/">Read more</a>]]></description>
				<content:encoded><![CDATA[
<p>It might be a bit confusing, because of historical reasons. </p>



<p>Metalab was initially developed as a desktop software to wrap up MetaProIQ workflow.  Upon more request, a cloud version was then developed for general users, called iMetaLab Cloud. Therefore Metalab (desktop) is the desktop version, while iMetalab (Cloud) is the cloud version. </p>



<p> We integrated an auto report module called iMetaReport into both desktop and cloud version for quick result review. </p>



<p>A suite of shiny apps, iMetaShiny, is all about the downstream data analysis and visualization.  </p>



<p>This iMetaWiki covers all the documents for all the software. </p>



<p>All these modules can be accessed by <a href="http://imetalab.ca">imetalab.ca</a></p>



<p></p>



<figure class="wp-block-image"><img src="https://ningzhibin.github.io/wp-content/uploads/2019/09/Version_2019_Family_banner-1024x739.png" alt="" class="wp-image-1191" srcset="https://ningzhibin.github.io/wp-content/uploads/2019/09/Version_2019_Family_banner-1024x739.png 1024w, https://ningzhibin.github.io/wp-content/uploads/2019/09/Version_2019_Family_banner-300x216.png 300w, https://ningzhibin.github.io/wp-content/uploads/2019/09/Version_2019_Family_banner-768x554.png 768w, https://ningzhibin.github.io/wp-content/uploads/2019/09/Version_2019_Family_banner-956x690.png 956w" sizes="(max-width: 1024px) 100vw, 1024px" /></figure>



<figure class="wp-block-image"><img src="https://ningzhibin.github.io/wp-content/uploads/2019/09/modules.png" alt="" class="wp-image-1196" srcset="https://ningzhibin.github.io/wp-content/uploads/2019/09/modules.png 632w, https://ningzhibin.github.io/wp-content/uploads/2019/09/modules-300x207.png 300w" sizes="(max-width: 632px) 100vw, 632px" /></figure>
]]></content:encoded>
			<wfw:commentRss>https://ningzhibin.github.io/2019/09/11/what-is-the-relationship-between-imetalab-members%ef%bc%9f/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Is iMetalab free?</title>
		<link>https://ningzhibin.github.io/2019/09/11/is-imetalab-free/</link>
		<comments>https://ningzhibin.github.io/2019/09/11/is-imetalab-free/#respond</comments>
		<pubDate>Wed, 11 Sep 2019 14:39:55 +0000</pubDate>
		<dc:creator><![CDATA[Zhibin]]></dc:creator>
				<category><![CDATA[FAQs]]></category>

		<guid isPermaLink="false">https://ningzhibin.github.io/?p=1187</guid>
		<description><![CDATA[The iMetalab platform (including Metalab Desktop, iMetalab Cloud, iMetaShiny apps) is completely free for academic and non-commercial users.&#160;For commercial use of iMetalab, please contact us to obtain a license.]]></description>
				<content:encoded><![CDATA[
<p> The iMetalab platform (including Metalab Desktop, iMetalab Cloud, iMetaShiny apps) is completely free for academic and non-commercial users.&nbsp;For commercial use of iMetalab, please contact us to obtain a license. </p>



<p></p>



<p></p>
]]></content:encoded>
			<wfw:commentRss>https://ningzhibin.github.io/2019/09/11/is-imetalab-free/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Experiment VS meta information</title>
		<link>https://ningzhibin.github.io/2019/09/08/experiment-vs-meta-information/</link>
		<comments>https://ningzhibin.github.io/2019/09/08/experiment-vs-meta-information/#respond</comments>
		<pubDate>Sun, 08 Sep 2019 18:36:14 +0000</pubDate>
		<dc:creator><![CDATA[Zhibin]]></dc:creator>
				<category><![CDATA[FAQs]]></category>
		<category><![CDATA[iMetaLab]]></category>
		<category><![CDATA[MetaLab]]></category>

		<guid isPermaLink="false">https://ningzhibin.github.io/?p=959</guid>
		<description><![CDATA[For example, we have a small project with two groups, 3 replicates each. We would like to set up 6 experiments, which will give us information about reproducibility. By doing so, you will be able to get 6 columns of...<br /><a class="read-more-button" href="https://ningzhibin.github.io/2019/09/08/experiment-vs-meta-information/">Read more</a>]]></description>
				<content:encoded><![CDATA[
<p> For example, we have a small project with two groups, 3 replicates each. We would like to set up 6 experiments, which will give us information about reproducibility. By doing so, you will be able to get 6 columns of quantification information. This is the Experiment information. We also need another grouping information, to tell the first 3 raw files/samples are from the control group, while the rest 3 files/samples are from the treatment group, to do the statistic analysis, e.g. looking for differentially expressed features among the two groups. </p>



<p> Meta info is not directly written/integrated into the result file, instead, it will be written into a separate .txt file, which will facilitate the <a href="https://ningzhibin.github.io/2019/09/06/new-html-based-report/">iMetaReport.</a></p>



<p>

The meta is optional, it does not affect anything during the automatic data analysis. You can add up to 5 groups of meta. Only the first groups of meta will be used for MetaReport (If there is meta info).

</p>
]]></content:encoded>
			<wfw:commentRss>https://ningzhibin.github.io/2019/09/08/experiment-vs-meta-information/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>An innovative way for experiment setting</title>
		<link>https://ningzhibin.github.io/2019/09/08/an-innovative-way-for-experiment-setting/</link>
		<comments>https://ningzhibin.github.io/2019/09/08/an-innovative-way-for-experiment-setting/#respond</comments>
		<pubDate>Sun, 08 Sep 2019 15:25:53 +0000</pubDate>
		<dc:creator><![CDATA[Zhibin]]></dc:creator>
				<category><![CDATA[iMetaLab]]></category>
		<category><![CDATA[MetaLab]]></category>
		<category><![CDATA[News & Updates]]></category>

		<guid isPermaLink="false">https://ningzhibin.github.io/?p=947</guid>
		<description><![CDATA[We have designed an effective way to set up the experiment. In MaxQuant, we have to fill Experiment column one-by-one or using Excel to edit the saved template and import it after. Here in MetaLab, you can just simply set...<br /><a class="read-more-button" href="https://ningzhibin.github.io/2019/09/08/an-innovative-way-for-experiment-setting/">Read more</a>]]></description>
				<content:encoded><![CDATA[
<p>We have designed an effective way to set up the experiment.  In MaxQuant, we have to fill Experiment column one-by-one or using Excel to edit the saved template and import it after. Here in MetaLab, you can just simply set the separator and choose the useful pieces of information to form the Experiment.  </p>



<figure class="wp-block-image"><img src="https://ningzhibin.github.io/wp-content/uploads/2019/09/experiment_setting-1-1024x266.png" alt="" class="wp-image-948" srcset="https://ningzhibin.github.io/wp-content/uploads/2019/09/experiment_setting-1-1024x266.png 1024w, https://ningzhibin.github.io/wp-content/uploads/2019/09/experiment_setting-1-300x78.png 300w, https://ningzhibin.github.io/wp-content/uploads/2019/09/experiment_setting-1-768x200.png 768w, https://ningzhibin.github.io/wp-content/uploads/2019/09/experiment_setting-1-956x248.png 956w, https://ningzhibin.github.io/wp-content/uploads/2019/09/experiment_setting-1.png 1189w" sizes="(max-width: 1024px) 100vw, 1024px" /></figure>



<p>

If you are not satisfied with the Experiment name set automatically, you are still able to edit in-cell by double-clicking it. 

</p>
]]></content:encoded>
			<wfw:commentRss>https://ningzhibin.github.io/2019/09/08/an-innovative-way-for-experiment-setting/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>New HTML based report</title>
		<link>https://ningzhibin.github.io/2019/09/06/new-html-based-report/</link>
		<comments>https://ningzhibin.github.io/2019/09/06/new-html-based-report/#comments</comments>
		<pubDate>Fri, 06 Sep 2019 21:16:27 +0000</pubDate>
		<dc:creator><![CDATA[Zhibin]]></dc:creator>
				<category><![CDATA[News & Updates]]></category>

		<guid isPermaLink="false">https://ningzhibin.github.io/?p=910</guid>
		<description><![CDATA[We have added a new function to provide a quick visualization of the result files. Once the Metalab task is done, a report is generated and will be opened in the browser, see the demo here More screenshot about peptide,...<br /><a class="read-more-button" href="https://ningzhibin.github.io/2019/09/06/new-html-based-report/">Read more</a>]]></description>
				<content:encoded><![CDATA[
<p>We have added a new function to provide a quick visualization of the result files. Once the Metalab task is done, a report is generated and will be opened in the browser, <a href="https://shiny.imetalab.ca/metalab_report/">see the demo here</a></p>



<figure class="wp-block-image"><img src="https://ningzhibin.github.io/wp-content/uploads/2019/09/report_index-1024x599.png" alt="" class="wp-image-911" srcset="https://ningzhibin.github.io/wp-content/uploads/2019/09/report_index-1024x599.png 1024w, https://ningzhibin.github.io/wp-content/uploads/2019/09/report_index-300x175.png 300w, https://ningzhibin.github.io/wp-content/uploads/2019/09/report_index-768x449.png 768w, https://ningzhibin.github.io/wp-content/uploads/2019/09/report_index-956x559.png 956w" sizes="(max-width: 1024px) 100vw, 1024px" /></figure>



<p>More screenshot about peptide, protein identification, taxon and function visualizations are as follows. </p>



<figure class="wp-block-image"><img src="https://ningzhibin.github.io/wp-content/uploads/2019/09/report_peptide-1024x787.png" alt="" class="wp-image-918" srcset="https://ningzhibin.github.io/wp-content/uploads/2019/09/report_peptide-1024x787.png 1024w, https://ningzhibin.github.io/wp-content/uploads/2019/09/report_peptide-300x231.png 300w, https://ningzhibin.github.io/wp-content/uploads/2019/09/report_peptide-768x590.png 768w, https://ningzhibin.github.io/wp-content/uploads/2019/09/report_peptide-956x735.png 956w, https://ningzhibin.github.io/wp-content/uploads/2019/09/report_peptide.png 1361w" sizes="(max-width: 1024px) 100vw, 1024px" /></figure>



<figure class="wp-block-image"><img src="https://ningzhibin.github.io/wp-content/uploads/2019/09/report_metamap-1024x764.png" alt="" class="wp-image-917" srcset="https://ningzhibin.github.io/wp-content/uploads/2019/09/report_metamap-1024x764.png 1024w, https://ningzhibin.github.io/wp-content/uploads/2019/09/report_metamap-300x224.png 300w, https://ningzhibin.github.io/wp-content/uploads/2019/09/report_metamap-768x573.png 768w, https://ningzhibin.github.io/wp-content/uploads/2019/09/report_metamap-956x713.png 956w, https://ningzhibin.github.io/wp-content/uploads/2019/09/report_metamap.png 1370w" sizes="(max-width: 1024px) 100vw, 1024px" /></figure>



<figure class="wp-block-image"><img src="https://ningzhibin.github.io/wp-content/uploads/2019/09/report_summary-1024x791.png" alt="" class="wp-image-916" srcset="https://ningzhibin.github.io/wp-content/uploads/2019/09/report_summary-1024x791.png 1024w, https://ningzhibin.github.io/wp-content/uploads/2019/09/report_summary-300x232.png 300w, https://ningzhibin.github.io/wp-content/uploads/2019/09/report_summary-768x593.png 768w, https://ningzhibin.github.io/wp-content/uploads/2019/09/report_summary-956x738.png 956w, https://ningzhibin.github.io/wp-content/uploads/2019/09/report_summary.png 1365w" sizes="(max-width: 1024px) 100vw, 1024px" /></figure>



<figure class="wp-block-image"><img src="https://ningzhibin.github.io/wp-content/uploads/2019/09/report_taxon-1024x788.png" alt="" class="wp-image-912" srcset="https://ningzhibin.github.io/wp-content/uploads/2019/09/report_taxon-1024x788.png 1024w, https://ningzhibin.github.io/wp-content/uploads/2019/09/report_taxon-300x231.png 300w, https://ningzhibin.github.io/wp-content/uploads/2019/09/report_taxon-768x591.png 768w, https://ningzhibin.github.io/wp-content/uploads/2019/09/report_taxon-956x735.png 956w, https://ningzhibin.github.io/wp-content/uploads/2019/09/report_taxon.png 1364w" sizes="(max-width: 1024px) 100vw, 1024px" /></figure>



<p>See<a href="https://shiny.imetalab.ca/metalab_report/"> the demo result report here</a></p>
]]></content:encoded>
			<wfw:commentRss>https://ningzhibin.github.io/2019/09/06/new-html-based-report/feed/</wfw:commentRss>
		<slash:comments>1</slash:comments>
		</item>
		<item>
		<title>Open-search for PTM now in Metalab</title>
		<link>https://ningzhibin.github.io/2019/09/05/open-search-for-ptm-now-in-metalab/</link>
		<comments>https://ningzhibin.github.io/2019/09/05/open-search-for-ptm-now-in-metalab/#respond</comments>
		<pubDate>Thu, 05 Sep 2019 02:43:06 +0000</pubDate>
		<dc:creator><![CDATA[Zhibin]]></dc:creator>
				<category><![CDATA[MetaLab]]></category>
		<category><![CDATA[News & Updates]]></category>
		<category><![CDATA[Related Publications]]></category>
		<category><![CDATA[opensearch]]></category>
		<category><![CDATA[PTM]]></category>

		<guid isPermaLink="false">https://ningzhibin.github.io/?p=565</guid>
		<description><![CDATA[Our work is just online in bioRxiv https://www.biorxiv.org/content/10.1101/753996v1 Abstract Studying the structure and function of microbiomes is an emerging research field. Metaproteomic approaches focusing on the characterization of expressed proteins and post-translational modifications (PTM) provide deeper understanding of microbial communities....<br /><a class="read-more-button" href="https://ningzhibin.github.io/2019/09/05/open-search-for-ptm-now-in-metalab/">Read more</a>]]></description>
				<content:encoded><![CDATA[
<p>Our work is just online in bioRxiv  <br><a href="https://www.biorxiv.org/content/10.1101/753996v1">https://www.biorxiv.org/content/10.1101/753996v1</a> </p>



<p></p>



<h2>Abstract</h2>



<p>Studying the structure and function of microbiomes is an emerging research field. Metaproteomic approaches focusing on the characterization of expressed proteins and post-translational modifications (PTM) provide deeper understanding of microbial communities. Previous research has highlighted the value of examining microbiome-wide protein expression in studying the roles of the microbiome in human diseases. Nevertheless, regulation of protein functions in complex microbiomes remains under-explored. This is mainly due to the lack of efficient bioinformatics tools to identify and quantify PTMs in the microbiome. Here we implemented an unrestricted open search strategy and developed an advanced multistage post-validation workflow for the identification of PTMs with high accuracy and sensitivity. This approach is applicable to proteomic and metaproteomic datasets. For metaproteomics, this method was combined into a complete data post-analysis workflow, including a quantitative module, to provide a novel and comprehensive data processing platform for metaproteomic analysis. This bioinformatics platform provides information about proteins, PTMs, taxa, functions, and pathways of microbial communities. The performance of the workflow was evaluated using conventional proteomics, metaproteomics from mouse and human gut microbiomes, and modification-specific enriched datasets.</p>
]]></content:encoded>
			<wfw:commentRss>https://ningzhibin.github.io/2019/09/05/open-search-for-ptm-now-in-metalab/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Checkpoints of MetaLab</title>
		<link>https://ningzhibin.github.io/2019/03/31/workflow-checkpoints-of-metalab/</link>
		<comments>https://ningzhibin.github.io/2019/03/31/workflow-checkpoints-of-metalab/#respond</comments>
		<pubDate>Sun, 31 Mar 2019 02:54:20 +0000</pubDate>
		<dc:creator><![CDATA[Zhibin]]></dc:creator>
				<category><![CDATA[Trouble shooting]]></category>
		<category><![CDATA[MetaLab]]></category>

		<guid isPermaLink="false">https://ningzhibin.github.io/?p=542</guid>
		<description><![CDATA[Get a better idea of how MetaLab works will help trouble shooting. For thermal Raw files convert raw files to .mgf you will be able to see .mgf files generated in the same directory of raw files mgf clustering to...<br /><a class="read-more-button" href="https://ningzhibin.github.io/2019/03/31/workflow-checkpoints-of-metalab/">Read more</a>]]></description>
				<content:encoded><![CDATA[<p>Get a better idea of how MetaLab works will help trouble shooting.</p>
<p>For thermal Raw files</p>
<ul>
<li>convert raw files to .mgf
<ul>
<li>you will be able to see .mgf files generated in the same directory of raw files</li>
</ul>
</li>
<li>mgf clustering to remove spectra redundancy to speed up the search</li>
<li>first round of database search against the &#8220;big&#8221; fasta database by Xtandem</li>
<li>extract the fasta from the Xtandem search result, to generate a sample specific database
<ul>
<li>you will be able see this file under output_folder/database/, a string of &#8220;sample_specific&#8221; will be appended to the original fasta file name.</li>
</ul>
</li>
<li>database search using the sample specific fasta database</li>
<li>taxon mapping</li>
<li>function mapping</li>
</ul>
]]></content:encoded>
			<wfw:commentRss>https://ningzhibin.github.io/2019/03/31/workflow-checkpoints-of-metalab/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
	</channel>
</rss>
