<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>Publications &#8211; iMetaWiki</title>
	<atom:link href="https://ningzhibin.github.io/tag/publications/feed/" rel="self" type="application/rss+xml" />
	<link>http://ningzhibin.github.io/</link>
	<description>All you need to know about the iMetaLab suite</description>
	<lastBuildDate>Mon, 27 Jan 2020 18:05:33 +0000</lastBuildDate>
	<language>en-US</language>
	<sy:updatePeriod>hourly</sy:updatePeriod>
	<sy:updateFrequency>1</sy:updateFrequency>
	<generator>https://wordpress.org/?v=5.0.3</generator>

<image>
	<url>https://ningzhibin.github.io/wp-content/uploads/2019/01/logo_M-1.png</url>
	<title>Publications &#8211; iMetaWiki</title>
	<link>http://ningzhibin.github.io/</link>
	<width>32</width>
	<height>32</height>
</image> 
	<item>
		<title>Metaproteomics reveals associations between microbiome and intestinal extracellular vesicle proteins in pediatric inflammatory bowel disease</title>
		<link>https://ningzhibin.github.io/2018/07/20/metaproteomics-reveals-associations-between-microbiome-and-intestinal-extracellular-vesicle-proteins-in-pediatric-inflammatory-bowel-disease/</link>
		<comments>https://ningzhibin.github.io/2018/07/20/metaproteomics-reveals-associations-between-microbiome-and-intestinal-extracellular-vesicle-proteins-in-pediatric-inflammatory-bowel-disease/#respond</comments>
		<pubDate>Fri, 20 Jul 2018 00:00:00 +0000</pubDate>
		<dc:creator><![CDATA[Zhang X, Deeke SA, Ning Z, Starr AE, Butcher J, Li J, Mayne J, Cheng K, Liao B, Li L, Singleton R, Mack D, Stintzi A, Figeys D]]></dc:creator>
				<category><![CDATA[Related Publications]]></category>
		<category><![CDATA[Publications]]></category>

		<guid isPermaLink="false">https://ningzhibin.github.io/2018/07/20/metaproteomics-reveals-associations-between-microbiome-and-intestinal-extracellular-vesicle-proteins-in-pediatric-inflammatory-bowel-disease/</guid>
		<description><![CDATA[Zhang X, Deeke SA, Ning Z, Starr AE, Butcher J, Li J, Mayne J, Cheng K, Liao B, Li L, Singleton R, Mack D, Stintzi A, Figeys D Nat Commun 2018 07;9(1):2873 PMID: 30030445 Abstract Alterations in gut microbiota have...<br /><a class="read-more-button" href="https://ningzhibin.github.io/2018/07/20/metaproteomics-reveals-associations-between-microbiome-and-intestinal-extracellular-vesicle-proteins-in-pediatric-inflammatory-bowel-disease/">Read more</a>]]></description>
				<content:encoded><![CDATA[<p>Zhang X, Deeke SA, Ning Z, Starr AE, Butcher J, Li J, Mayne J, Cheng K, Liao B, Li L, Singleton R, Mack D, Stintzi A, Figeys D</p>
<p>Nat Commun 2018 07;9(1):2873</p>
<p>PMID: <a href='https://www.ncbi.nlm.nih.gov/pubmed/30030445' title=''>30030445</a></a></p>
<h2>Abstract</h2>
<p><p>Alterations in gut microbiota have been implicated in the pathogenesis of inflammatory bowel disease (IBD), however factors that mediate the host-microbiota interactions remain largely unknown. Here we collected mucosal-luminal interface samples from a pediatric IBD inception cohort and characterized both the human and microbiota proteins using metaproteomics. We show that microbial proteins related to oxidative stress responses are upregulated in IBD cases compared to controls. In particular, we demonstrate that the expression of human proteins related to oxidative antimicrobial activities is increased in IBD cases and correlates with the alteration of microbial functions. Additionally, we reveal that many of these human proteins are present and show altered abundance in isolated free extracellular vesicles (EVs). Therefore, our study suggests that the alteration of intestinal EV proteomes is associated with the aberrant host-microbiota interactions in IBD.</p></p>
<div class="post-views post-521 entry-meta">
			<span class="post-views-icon dashicons dashicons-chart-bar"></span>
			<span class="post-views-label">Page Views: </span>
			<span class="post-views-count">20</span>
			</div>]]></content:encoded>
			<wfw:commentRss>https://ningzhibin.github.io/2018/07/20/metaproteomics-reveals-associations-between-microbiome-and-intestinal-extracellular-vesicle-proteins-in-pediatric-inflammatory-bowel-disease/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>MetaLab: an automated pipeline for metaproteomic data analysis</title>
		<link>https://ningzhibin.github.io/2017/12/02/metalab-an-automated-pipeline-for-metaproteomic-data-analysis/</link>
		<comments>https://ningzhibin.github.io/2017/12/02/metalab-an-automated-pipeline-for-metaproteomic-data-analysis/#respond</comments>
		<pubDate>Sat, 02 Dec 2017 00:00:00 +0000</pubDate>
		<dc:creator><![CDATA[Cheng K, Ning Z, Zhang X, Li L, Liao B, Mayne J, Stintzi A, Figeys D]]></dc:creator>
				<category><![CDATA[Related Publications]]></category>
		<category><![CDATA[Publications]]></category>

		<guid isPermaLink="false">https://ningzhibin.github.io/2017/12/02/metalab-an-automated-pipeline-for-metaproteomic-data-analysis/</guid>
		<description><![CDATA[Cheng K, Ning Z, Zhang X, Li L, Liao B, Mayne J, Stintzi A, Figeys D Microbiome 2017 Dec;5(1):157 PMID: 29197424 Abstract BACKGROUND: Research involving microbial ecosystems has drawn increasing attention in recent years. Studying microbe-microbe, host-microbe, and environment-microbe interactions...<br /><a class="read-more-button" href="https://ningzhibin.github.io/2017/12/02/metalab-an-automated-pipeline-for-metaproteomic-data-analysis/">Read more</a>]]></description>
				<content:encoded><![CDATA[<p>Cheng K, Ning Z, Zhang X, Li L, Liao B, Mayne J, Stintzi A, Figeys D</p>
<p>Microbiome 2017 Dec;5(1):157</p>
<p>PMID: <a href='https://www.ncbi.nlm.nih.gov/pubmed/29197424' title=''>29197424</a></a></p>
<h2>Abstract</h2>
<p><p><strong>BACKGROUND: </strong>Research involving microbial ecosystems has drawn increasing attention in recent years. Studying microbe-microbe, host-microbe, and environment-microbe interactions are essential for the understanding of microbial ecosystems. Currently, metaproteomics provide qualitative and quantitative information of proteins, providing insights into the functional changes of microbial communities. However, computational analysis of large-scale data generated in metaproteomic studies remains a challenge. Conventional proteomic software have difficulties dealing with the extreme complexity and species diversity present in microbiome samples leading to lower rates of peptide and protein identification. To address this issue, we previously developed the MetaPro-IQ approach for highly efficient microbial protein/peptide identification and quantification.</p>
<p><strong>RESULT: </strong>Here, we developed an integrated software platform, named MetaLab, providing a complete and automated, user-friendly pipeline for fast microbial protein identification, quantification, as well as taxonomic profiling, directly from mass spectrometry raw data. Spectral clustering adopted in the pre-processing step dramatically improved the speed of peptide identification from database searches. Quantitative information of identified peptides was used for estimating the relative abundance of taxa at all phylogenetic ranks. Taxonomy result files exported by MetaLab are fully compatible with widely used metagenomics tools. Herein, the potential of MetaLab is evaluated by reanalyzing a metaproteomic dataset from mouse gut microbiome samples.</p>
<p><strong>CONCLUSION: </strong>MetaLab is a fully automatic software platform enabling an integrated data-processing pipeline for metaproteomics. The function of sample-specific database generation can be very advantageous for searching peptides against huge protein databases. It provides a seamless connection between peptide determination and taxonomic profiling; therefore, the peptide abundance is readily used for measuring the microbial variations. MetaLab is designed as a versatile, efficient, and easy-to-use tool which can greatly simplify the procedure of metaproteomic data analysis for researchers in microbiome studies.</p></p>
<div class="post-views post-519 entry-meta">
			<span class="post-views-icon dashicons dashicons-chart-bar"></span>
			<span class="post-views-label">Page Views: </span>
			<span class="post-views-count">26</span>
			</div>]]></content:encoded>
			<wfw:commentRss>https://ningzhibin.github.io/2017/12/02/metalab-an-automated-pipeline-for-metaproteomic-data-analysis/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Evaluating in Vitro Culture Medium of Gut Microbiome with Orthogonal Experimental Design and a Metaproteomics Approach</title>
		<link>https://ningzhibin.github.io/2017/11/22/evaluating-in-vitro-culture-medium-of-gut-microbiome-with-orthogonal-experimental-design-and-a-metaproteomics-approach/</link>
		<comments>https://ningzhibin.github.io/2017/11/22/evaluating-in-vitro-culture-medium-of-gut-microbiome-with-orthogonal-experimental-design-and-a-metaproteomics-approach/#respond</comments>
		<pubDate>Wed, 22 Nov 2017 00:00:00 +0000</pubDate>
		<dc:creator><![CDATA[Li L, Zhang X, Ning Z, Mayne J, Moore JI, Butcher J, Chiang CK, Mack D, Stintzi A, Figeys D]]></dc:creator>
				<category><![CDATA[Related Publications]]></category>
		<category><![CDATA[Publications]]></category>

		<guid isPermaLink="false">https://ningzhibin.github.io/2017/11/22/evaluating-in-vitro-culture-medium-of-gut-microbiome-with-orthogonal-experimental-design-and-a-metaproteomics-approach/</guid>
		<description><![CDATA[Li L, Zhang X, Ning Z, Mayne J, Moore JI, Butcher J, Chiang CK, Mack D, Stintzi A, Figeys D J. Proteome Res. 2018 01;17(1):154-163 PMID: 29130306 Abstract In vitro culture based approaches are time- and cost-effective solutions for rapidly...<br /><a class="read-more-button" href="https://ningzhibin.github.io/2017/11/22/evaluating-in-vitro-culture-medium-of-gut-microbiome-with-orthogonal-experimental-design-and-a-metaproteomics-approach/">Read more</a>]]></description>
				<content:encoded><![CDATA[<p>Li L, Zhang X, Ning Z, Mayne J, Moore JI, Butcher J, Chiang CK, Mack D, Stintzi A, Figeys D</p>
<p>J. Proteome Res. 2018 01;17(1):154-163</p>
<p>PMID: <a href='https://www.ncbi.nlm.nih.gov/pubmed/29130306' title=''>29130306</a></a></p>
<h2>Abstract</h2>
<p><p>In vitro culture based approaches are time- and cost-effective solutions for rapidly evaluating the effects of drugs or natural compounds against microbiomes. The nutritional composition of the culture medium is an important determinant for effectively maintaining the gut microbiome in vitro. This study combines orthogonal experimental design and a metaproteomics approach to obtaining functional insights into the effects of different medium components on the microbiome. Our results show that the metaproteomic profile respond differently to medium components, including inorganic salts, bile salts, mucin, and short-chain fatty acids. Multifactor analysis of variance further revealed significant main and interaction effects of inorganic salts, bile salts, and mucin on the different functional groups of gut microbial proteins. While a broad regulating effect was observed on basic metabolic pathways, different medium components also showed significant modulations on cell wall, membrane, and envelope biogenesis and cell motility related functions. In particular, flagellar assembly related proteins were significantly responsive to the presence of mucin. This study provides information on the functional influences of medium components on the in vitro growth of microbiome communities and gives insight on the key components that must be considered when selecting and optimizing media for culturing ex vivo microbiotas.</p></p>
<div class="post-views post-522 entry-meta">
			<span class="post-views-icon dashicons dashicons-chart-bar"></span>
			<span class="post-views-label">Page Views: </span>
			<span class="post-views-count">20</span>
			</div>]]></content:encoded>
			<wfw:commentRss>https://ningzhibin.github.io/2017/11/22/evaluating-in-vitro-culture-medium-of-gut-microbiome-with-orthogonal-experimental-design-and-a-metaproteomics-approach/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Proteomic and Metaproteomic Approaches to Understand Host-Microbe Interactions</title>
		<link>https://ningzhibin.github.io/2017/11/09/proteomic-and-metaproteomic-approaches-to-understand-host-microbe-interactions/</link>
		<comments>https://ningzhibin.github.io/2017/11/09/proteomic-and-metaproteomic-approaches-to-understand-host-microbe-interactions/#respond</comments>
		<pubDate>Thu, 09 Nov 2017 00:00:00 +0000</pubDate>
		<dc:creator><![CDATA[Starr AE, Deeke SA, Li L, Zhang X, Daoud R, Ryan J, Ning Z, Cheng K, Nguyen LVH, Abou-Samra E, Lavallée-Adam M, Figeys D]]></dc:creator>
				<category><![CDATA[Related Publications]]></category>
		<category><![CDATA[Publications]]></category>

		<guid isPermaLink="false">https://ningzhibin.github.io/2017/11/09/proteomic-and-metaproteomic-approaches-to-understand-host-microbe-interactions/</guid>
		<description><![CDATA[Starr AE, Deeke SA, Li L, Zhang X, Daoud R, Ryan J, Ning Z, Cheng K, Nguyen LVH, Abou-Samra E, Lavallée-Adam M, Figeys D Anal. Chem. 2018 01;90(1):86-109 PMID: 29061041 Abstract Page Views: 21]]></description>
				<content:encoded><![CDATA[<p>Starr AE, Deeke SA, Li L, Zhang X, Daoud R, Ryan J, Ning Z, Cheng K, Nguyen LVH, Abou-Samra E, Lavallée-Adam M, Figeys D</p>
<p>Anal. Chem. 2018 01;90(1):86-109</p>
<p>PMID: <a href='https://www.ncbi.nlm.nih.gov/pubmed/29061041' title=''>29061041</a></a></p>
<h2>Abstract</h2></p>
<div class="post-views post-523 entry-meta">
			<span class="post-views-icon dashicons dashicons-chart-bar"></span>
			<span class="post-views-label">Page Views: </span>
			<span class="post-views-count">21</span>
			</div>]]></content:encoded>
			<wfw:commentRss>https://ningzhibin.github.io/2017/11/09/proteomic-and-metaproteomic-approaches-to-understand-host-microbe-interactions/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Deep Metaproteomics Approach for the Study of Human Microbiomes</title>
		<link>https://ningzhibin.github.io/2017/08/11/deep-metaproteomics-approach-for-the-study-of-human-microbiomes/</link>
		<comments>https://ningzhibin.github.io/2017/08/11/deep-metaproteomics-approach-for-the-study-of-human-microbiomes/#respond</comments>
		<pubDate>Fri, 11 Aug 2017 00:00:00 +0000</pubDate>
		<dc:creator><![CDATA[Zhang X, Chen W, Ning Z, Mayne J, Mack D, Stintzi A, Tian R, Figeys D]]></dc:creator>
				<category><![CDATA[Related Publications]]></category>
		<category><![CDATA[Publications]]></category>

		<guid isPermaLink="false">https://ningzhibin.github.io/2017/08/11/deep-metaproteomics-approach-for-the-study-of-human-microbiomes/</guid>
		<description><![CDATA[Zhang X, Chen W, Ning Z, Mayne J, Mack D, Stintzi A, Tian R, Figeys D Anal. Chem. 2017 Sep;89(17):9407-9415 PMID: 28749657 Abstract Host-microbiome interactions have been shown to play important roles in human health and diseases. Most of the...<br /><a class="read-more-button" href="https://ningzhibin.github.io/2017/08/11/deep-metaproteomics-approach-for-the-study-of-human-microbiomes/">Read more</a>]]></description>
				<content:encoded><![CDATA[<p>Zhang X, Chen W, Ning Z, Mayne J, Mack D, Stintzi A, Tian R, Figeys D</p>
<p>Anal. Chem. 2017 Sep;89(17):9407-9415</p>
<p>PMID: <a href='https://www.ncbi.nlm.nih.gov/pubmed/28749657' title=''>28749657</a></a></p>
<h2>Abstract</h2>
<p><p>Host-microbiome interactions have been shown to play important roles in human health and diseases. Most of the current studies of the microbiome have been performed by genomic approaches through next-generation sequencing. Technologies, such as metaproteomics, for functional analysis of the microbiome are needed to better understand the intricate host-microbiome interactions. However, significant efforts to improve the depth and resolution of gut metaproteomics are still required. In this study, we combined an efficient sample preparation technique, high resolution mass spectrometry, and metaproteomic bioinformatics tools to perform ultradeep metaproteomic analysis of human gut microbiome from stool. We reported the deepest analysis of the microbiome to date with an average of 20 558 protein groups identified per sample analysis. Moreover, strain resolution taxonomic and pathway analysis using deep metaproteomics revealed strain level variations, in particular for Faecalibacterium prausnitzii, in the microbiome from the different individuals. We also reported that the human proteins identified in stool samples are functionally enriched in extracellular region pathways and in particular those proteins involved in defense response against microbial organisms. Deep metaproteomics is a promising approach to perform in-depth microbiome analysis and simultaneously reveals both human and microbial changes that are not readily apparent using the standard genomic approaches.</p></p>
<div class="post-views post-524 entry-meta">
			<span class="post-views-icon dashicons dashicons-chart-bar"></span>
			<span class="post-views-label">Page Views: </span>
			<span class="post-views-count">20</span>
			</div>]]></content:encoded>
			<wfw:commentRss>https://ningzhibin.github.io/2017/08/11/deep-metaproteomics-approach-for-the-study-of-human-microbiomes/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Assessing the impact of protein extraction methods for human gut metaproteomics</title>
		<link>https://ningzhibin.github.io/2017/07/10/assessing-the-impact-of-protein-extraction-methods-for-human-gut-metaproteomics/</link>
		<comments>https://ningzhibin.github.io/2017/07/10/assessing-the-impact-of-protein-extraction-methods-for-human-gut-metaproteomics/#respond</comments>
		<pubDate>Mon, 10 Jul 2017 00:00:00 +0000</pubDate>
		<dc:creator><![CDATA[Zhang X, Li L, Mayne J, Ning Z, Stintzi A, Figeys D]]></dc:creator>
				<category><![CDATA[Related Publications]]></category>
		<category><![CDATA[Publications]]></category>

		<guid isPermaLink="false">https://ningzhibin.github.io/2017/07/10/assessing-the-impact-of-protein-extraction-methods-for-human-gut-metaproteomics/</guid>
		<description><![CDATA[Zhang X, Li L, Mayne J, Ning Z, Stintzi A, Figeys D J Proteomics 2018 May;180:120-127 PMID: 28705725 Abstract Metaproteomics is a promising methodology for the functional characterizations of the gut microbiome. However, the performance of metaproteomic analysis is affected...<br /><a class="read-more-button" href="https://ningzhibin.github.io/2017/07/10/assessing-the-impact-of-protein-extraction-methods-for-human-gut-metaproteomics/">Read more</a>]]></description>
				<content:encoded><![CDATA[<p>Zhang X, Li L, Mayne J, Ning Z, Stintzi A, Figeys D</p>
<p>J Proteomics 2018 May;180:120-127</p>
<p>PMID: <a href='https://www.ncbi.nlm.nih.gov/pubmed/28705725' title=''>28705725</a></a></p>
<h2>Abstract</h2>
<p><p>Metaproteomics is a promising methodology for the functional characterizations of the gut microbiome. However, the performance of metaproteomic analysis is affected by protein extraction protocols in terms of the amount of protein recovered and the relative abundance of different bacteria observed in microbiome. Currently, there is a lack of consistency on protein extraction methods in published metaproteomics studies. Here we evaluated the effects of different protein extraction methods on human fecal metaproteome characterizations. We found that sodium dodecyl sulfate (SDS)-based lysis buffer obtained higher protein yields and peptide/protein group identifications compared to urea and the non-ionic detergent-based B-Per buffer. The addition of bead beating to any of the extraction buffers increased both protein yields and protein identifications. As well, bead beating led to a significant increase of the relative abundances of Firmicutes and Actinobacteria. We also demonstrated that ultrasonication, another commonly used mechanical disruption approach, performed even better than bead beating for gut microbial protein extractions. Importantly, proteins of the basic metabolic pathways showed significantly higher relative abundances when using ultrasonication. Overall, these results demonstrate that protein extraction protocols markedly impact the metaproteomic results and recommend a protein extraction protocol with both SDS and ultrasonication for metaproteomic studies.</p>
<p><strong>BIOLOGICAL SIGNIFICANCE: </strong>The gut microbiome is emerging as an important factor influencing human health. Metaproteomics is promising for advancing the understanding of the functional roles of the microbiome in disease. However, metaproteomics suffers from a lack of consistent sample preparation procedures. In the present study, protein extraction protocols for fecal microbiome samples were evaluated for their effects on protein yields, peptide identifications, protein group identifications, taxonomic compositions and functional category distributions. While different protocols favor different microbial taxa and protein functions, our results suggest that a protein extraction protocol using sodium dodecyl sulfate (SDS) and ultrasonication provides the best performance for general shotgun metaproteomics studies.</p></p>
<div class="post-views post-525 entry-meta">
			<span class="post-views-icon dashicons dashicons-chart-bar"></span>
			<span class="post-views-label">Page Views: </span>
			<span class="post-views-count">19</span>
			</div>]]></content:encoded>
			<wfw:commentRss>https://ningzhibin.github.io/2017/07/10/assessing-the-impact-of-protein-extraction-methods-for-human-gut-metaproteomics/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>MetaPro-IQ: a universal metaproteomic approach to studying human and mouse gut microbiota</title>
		<link>https://ningzhibin.github.io/2016/06/24/metapro-iq-a-universal-metaproteomic-approach-to-studying-human-and-mouse-gut-microbiota/</link>
		<comments>https://ningzhibin.github.io/2016/06/24/metapro-iq-a-universal-metaproteomic-approach-to-studying-human-and-mouse-gut-microbiota/#respond</comments>
		<pubDate>Fri, 24 Jun 2016 00:00:00 +0000</pubDate>
		<dc:creator><![CDATA[Zhang X, Ning Z, Mayne J, Moore JI, Li J, Butcher J, Deeke SA, Chen R, Chiang CK, Wen M, Mack D, Stintzi A, Figeys D]]></dc:creator>
				<category><![CDATA[Related Publications]]></category>
		<category><![CDATA[Publications]]></category>

		<guid isPermaLink="false">https://ningzhibin.github.io/2016/06/24/metapro-iq-a-universal-metaproteomic-approach-to-studying-human-and-mouse-gut-microbiota/</guid>
		<description><![CDATA[Zhang X, Ning Z, Mayne J, Moore JI, Li J, Butcher J, Deeke SA, Chen R, Chiang CK, Wen M, Mack D, Stintzi A, Figeys D Microbiome 2016 06;4(1):31 PMID: 27343061 Abstract BACKGROUND: The gut microbiota has been shown to...<br /><a class="read-more-button" href="https://ningzhibin.github.io/2016/06/24/metapro-iq-a-universal-metaproteomic-approach-to-studying-human-and-mouse-gut-microbiota/">Read more</a>]]></description>
				<content:encoded><![CDATA[<p>Zhang X, Ning Z, Mayne J, Moore JI, Li J, Butcher J, Deeke SA, Chen R, Chiang CK, Wen M, Mack D, Stintzi A, Figeys D</p>
<p>Microbiome 2016 06;4(1):31</p>
<p>PMID: <a href='https://www.ncbi.nlm.nih.gov/pubmed/27343061' title=''>27343061</a></a></p>
<h2>Abstract</h2>
<p><p><strong>BACKGROUND: </strong>The gut microbiota has been shown to be closely associated with human health and disease. While next-generation sequencing can be readily used to profile the microbiota taxonomy and metabolic potential, metaproteomics is better suited for deciphering microbial biological activities. However, the application of gut metaproteomics has largely been limited due to the low efficiency of protein identification. Thus, a high-performance and easy-to-implement gut metaproteomic approach is required.</p>
<p><strong>RESULTS: </strong>In this study, we developed a high-performance and universal workflow for gut metaproteome identification and quantification (named MetaPro-IQ) by using the close-to-complete human or mouse gut microbial gene catalog as database and an iterative database search strategy. An average of 38 and 33 % of the acquired tandem mass spectrometry (MS) spectra was confidently identified for the studied mouse stool and human mucosal-luminal interface samples, respectively. In total, we accurately quantified 30,749 protein groups for the mouse metaproteome and 19,011 protein groups for the human metaproteome. Moreover, the MetaPro-IQ approach enabled comparable identifications with the matched metagenome database search strategy that is widely used but needs prior metagenomic sequencing. The response of gut microbiota to high-fat diet in mice was then assessed, which showed distinct metaproteome patterns for high-fat-fed mice and identified 849 proteins as significant responders to high-fat feeding in comparison to low-fat feeding.</p>
<p><strong>CONCLUSIONS: </strong>We present MetaPro-IQ, a metaproteomic approach for highly efficient intestinal microbial protein identification and quantification, which functions as a universal workflow for metaproteomic studies, and will thus facilitate the application of metaproteomics for better understanding the functions of gut microbiota in health and disease.</p></p>
<div class="post-views post-520 entry-meta">
			<span class="post-views-icon dashicons dashicons-chart-bar"></span>
			<span class="post-views-label">Page Views: </span>
			<span class="post-views-count">20</span>
			</div>]]></content:encoded>
			<wfw:commentRss>https://ningzhibin.github.io/2016/06/24/metapro-iq-a-universal-metaproteomic-approach-to-studying-human-and-mouse-gut-microbiota/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
	</channel>
</rss>
